website/projects/awssync.py

```diff
@@ -22,8 +26,37 @@ def button_pressed(self):
         :return: True if function executes successfully
         """
         self.logger.info("Pressed button")
```

Comment:
Please add line `self.logger.setLevel(logging.DEBUG)` right after the logger is initialized, so all logs go to `stdout` which is useful for all of us during development.